src/components/backend-ai-resource-broker.ts

```diff
@@ -601,7 +603,51 @@ export default class BackendAiResourceBroker extends BackendAIPage {
       }
 
       this.total_slot = total_slot;
-      this.total_resource_group_slot = total_resource_group_slot;
+      if (!globalThis.backendaiclient._config.hideAgents) {
+        const status = 'ALIVE';
+        const limit = 20;
```

Comment:
What happens when there are more than 20 Agents?

Reply:
I'm afraid that didn't test it yet.
For now, I just referenced the basic parameter value from [here](https://github.com/lablup/backend.ai-webui/blob/e4949338a3cf561b10cc606e963d770d41992c15/src/components/backend-ai-agent-summary-list.ts#L203-L207).
If the value needs adjustment following code should be changed accordingly.

Reply:
Temporary follow-up PR: https://github.com/lablup/backend.ai-webui/pull/1599.
Please review and merge if it looks okay. @lizable

Reply:
Merged!